[check stability] Remove Chrome bug workaround

jugglinmike opened the pull request:
The non-deterministic behavior that was previously causing instability
in Chrome test runs has recently been corrected in the Chrome project
itself [1]. This obviates the need for a local workaround. Revert the
workaround, but continue to allow failures in the continuous integration
environment so that this change may be validated.

This patch is the result of reverting two commits:

- Revert "[check stability] Apply DBUS configuration in C.I."
  This reverts commit 91c54d13629a66a3d5b8817fdcd8278a152871cb.
- Revert "Ensure dbus is running when stability check runs in Chrome."
  This reverts commit 0509dfe88e8209a0983b08eed655dfe52804fec2.

[1] https://bugs.chromium.org/p/chromium/issues/detail?id=713947#c13

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/w3c/web-platform-tests/6064)
<!-- Reviewable:end -->

jugglinmike committed:
[check stability] Remove Chrome bug workaround

The non-deterministic behavior that was previously causing instability
in Chrome test runs has recently been corrected in the Chrome project
itself [1]. This obviates the need for a local workaround. Revert the
workaround, but continue to allow failures in the continuous integration
environment so that this change may be validated.

This patch is the result of reverting two commits:

- Revert "[check stability] Apply DBUS configuration in C.I."
  This reverts commit 91c54d13629a66a3d5b8817fdcd8278a152871cb.
- Revert "Ensure dbus is running when stability check runs in Chrome."
  This reverts commit 0509dfe88e8209a0983b08eed655dfe52804fec2.

[1] https://bugs.chromium.org/p/chromium/issues/detail?id=713947#c13

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/236417136)

# Firefox (nightly) #
Testing web-platform-tests at revision d41a22d6819fc4216119376f0f079542e6c28d92
Using browser at version BuildID 20170524100215; SourceStamp 291a11111bdd05c5cd55dd552da4b1285ceba9b2
Starting 10 test iterations
No tests run.

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/236417137)

# Sauce (safari) #
Testing web-platform-tests at revision d41a22d6819fc4216119376f0f079542e6c28d92
Using browser at version 10.0
Starting 10 test iterations
No tests run.

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/236417143)

# Chrome (unstable) #
Testing web-platform-tests at revision d41a22d6819fc4216119376f0f079542e6c28d92
Using browser at version 60.0.3107.4 dev
Starting 10 test iterations
No tests run.

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/236417144)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision d41a22d6819fc4216119376f0f079542e6c28d92

jgraham commented on check_stability.py:
```diff
@@ -1,6 +1,7 @@
 from __future__ import print_function
 
 import argparse
+from ConfigParser import SafeConfigParser
```
Why did this move? Convention is that `from…` imports go after plain `import` and that sorting is by unicode codepoint.

jugglinmike replied:
This change set reverts 0509dfe88e8209a0983b08eed655dfe52804fec2, which seems to have included an unrelated change. I'll push up a second commit that is explicitly dedicated to honoring to the convention you've referenced.

jugglinmike committed:
[check stability] Reformat according to convention

Code style convention in this project dicatates that `from...` import
statements follow "plain" `import` statments and within that ordering,
sorting is by unicode codepoint.
